ms/auth.json

```diff
@@ -0,0 +1,274 @@
+      "email_input_error_required": "E-mel diperlukan",
+      "email_input_error_duplicate_email_found": "E-mel ini telah melanggan",
+      "continue_button": "Daftar",
+      "success_heading": "Berjaya! Selamat datang.",
+      "credit_powered_by": "Dikuasakan oleh"
+    },
+    "password_verify_page": {
+      "page_title": "Sahkan kata laluan | ${business_name}",
+      "heading": "Masukkan kata laluan anda",
+      "description": "Masukkan kata laluan anda untuk ${email_address}",
+      "password_input_label": "Kata laluan",
+      "password_input_error_required": "Masukkan kata laluan anda",
+      "password_input_error_incorrect": "Masukkan kata laluan yang sah",
+      "password_input_error_too_short": "Kata laluan perlu mengandungi sekurang-kurangnya 8 aksara",
+      "continue_button": "Teruskan",
+      "cant_sign_in_description": "Can't sign in?",
+      "cant_sign_in_link": "Reset password",
+      "password_input_error_unknown_error": "Ralat dengan pengesahan"
+    },
+    "sms_verify_code_page": {
+      "page_title": "Pengesahan berbilang faktor | ${business_name}",
+      "heading": "Sahkan nombor telefon",
+      "description": "Masukkan kod yang baru sahaja kami hantar ke ${phone_number}",
+      "code_input_label": "Kod",
+      "code_input_error_expired_code": "Kod pengesahan anda telah tamat tempoh, sila minta yg baru",
+      "code_input_error_incorrect_code": "Sila masukkan kod pengesahan yang sah",
+      "code_input_error_incorrect_code_length": "Kod pengesahan mestilah mengandungi 6 digit",
+      "code_input_error_resend_requested_too_soon": "Tidak dapat menghantar kod yg lain buat masa sekarang. Sila cuba sebentar lagi.",
+      "confirmation_code_input_error_unknown_error": "Ralat dalam proses pengesahan",
+      "continue_button": "Sahkan",
+      "didnt_receive_a_code_description": "Tidak menerima sebarang kod?",
+      "didnt_receive_a_code_link": "Hantar kod semula",
+      "choose_another_method_link": "Sila pilih kaedah yang lain"
+    },
+    "recovery_code_verify_page": {
+      "page_title": "Gunakan kod pemulihan | ${business_name}",
+      "heading": "Kod pemulihan",
+      "description": "Masukkan salah satu dari kod pemulihan yang telah diberikan semasa proses pendaftaran awal.",
+      "recovery_code_input_label": "Kod pemulihan",
+      "continue_button": "Teruskan",
+      "choose_another_method_link": "Sila pilih kaedah yang lain",
+      "recovery_code_input_error_unknown_error": "Terdapat masalah membuat pengesahan dengan kod pemulihan ini",
+      "recovery_code_input_error_required": "Masukkan kod pemulihan anda",
+      "recovery_code_input_error_too_short": "Kod pemulihan perlu mengandungi sekurang-kurangnya 9 aksara",
+      "recovery_code_input_error_invalid": "Masukkan kod pemulihan yang sah"
+    },
+    "verification_code_sms": {
+      "description": "${generated_code} adalah kod sekali guna anda untuk mendaftar masuk ke ${business_name} @${receiver_domain} #${generated_code}"
```

Comment:
Thanks for the PR @kotaksempit can we add two `\n` so a new line is formed in the SMS 
```
"${generated_code} adalah kod sekali guna anda untuk mendaftar masuk ke ${business_name}\n\n@${receiver_domain} #${generated_code}"
```

Reply:
Noted.